hbp_knowledge/proteostasis/Brekk2018.bel

```diff
@@ -73,123 +73,149 @@ DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifact
 SET Citation = {"PubMed", "30562574"}
 
 SET Evidence = "Chaperone-mediated autophagy (CMA) is a selective pathway for lysosomal degradation of long-lived cytosolic proteins, and differs from the other two modes of cytosol-to-lysosome targeting, macro- and micro-autophagy, in that it 1) does not involve vesicle formation, and 2) that specific interactions with a complex of molecular chaperones must occur prior to direct translocation of the substrate protein through the lysosomal membrane (Cuervo 2011)."
-
+SET Confidence = "High"
 bp(GO:"chaperone-mediated autophagy") -> deg(a(MESH:Proteins,loc(GO:lysosome)))
 bp(GO:macroautophagy) -> tloc(a(MESH:"Cytoplasmic Vesicles"),fromLoc(GO:cytoplasm),toLoc(GO:lysosome))
 bp(GO:"lysosomal microautophagy") -> tloc(a(MESH:"Cytoplasmic Vesicles"),fromLoc(GO:cytoplasm),toLoc(GO:lysosome))
 composite(a(MESH:Proteins),a(MESH:"Molecular Chaperones")) -> tloc(a(MESH:Proteins),fromLoc(GO:cytoplasm),toLoc(GO:lysosome))
+UNSET Confidence
 
 SET Evidence = "Once bound by the chaperone complex, substrates are directly unfolded and translocated through the lysosomal membrane by a multimeric channel consisting of the lysosome-associated membrane protein type 2A (LAMP2A)"
-
+SET Confidence = "Medium"
+composite(a(MESH:Proteins),a(MESH:"Molecular Chaperones")) -> path(MESH:"Protein Unfolding")
 p(HGNC:LAMP2,loc(GO:"lysosomal membrane")) -> tloc(a(MESH:Proteins),fromLoc(GO:cytoplasm),toLoc(GO:lysosome))
+UNSET Confidence
 
-SET Evidence = " Furthermore, LAMP2A acts as the rate-limiting step for the activity of the CMA pathway, with rate of the assembly / disassembly of the LAMP2A complex in the lysosomal membrane being paramount for its function (Cuervo and Dice 1996;Bandyopadhyay et al. 2008)."
 
+SET Evidence = " Furthermore, LAMP2A acts as the rate-limiting step for the activity of the CMA pathway, with rate of the assembly / disassembly of the LAMP2A complex in the lysosomal membrane being paramount for its function (Cuervo and Dice 1996;Bandyopadhyay et al. 2008)."
+SET Confidence = "Low"
 p(HGNC:LAMP2) reg bp(GO:"chaperone-mediated autophagy")
+#Can you let me know which statement shows about the next two Statements
```

Comment:
They are on regard to the statement "rate of the assembly / disassembly of the LAMP2A complex in the lysosomal membrane being paramount for its function".  Disassembly brings LAMP2 in cytoplasm and CAM is blocked while assembly puts LAMP2 on lysosomal membrane and CAM is activated.